packages/vm/src/imports.rs

```diff
@@ -18,12 +21,16 @@ use crate::errors::{CommunicationError, VmError, VmResult};
 #[cfg(feature = "iterator")]
 use crate::memory::maybe_read_region;
 use crate::memory::{read_region, write_region};
+use crate::sections::decode_sections;
+#[allow(unused_imports)]
+use crate::sections::encode_sections;
 use crate::serde::to_vec;
 use crate::GasInfo;
 
 const GAS_COST_SECP256K1_VERIFY_SIGNATURE: u64 = 100;
 const GAS_COST_SECP256K1_RECOVER_PUBKEY_SIGNATURE: u64 = 100;
 const GAS_COST_VERIFY_ED25519_SIGNATURE: u64 = 100;
+const GAS_COST_BATCH_VERIFY_ED25519_SIGNATURE: u64 = 80;
```

Comment:
Notes:

- Batch cost can probably be derived from / based on the gas cost of the non-batch method, divided by a number between 2 and 4 (usually known as 3 :-). Based on the graph in https://docs.rs/ed25519-zebra/2.2.0/ed25519_zebra/batch/index.html.
- We'll do some measurements / comparisons to confirm.

- Final gas cost will have to be computed by multiplying this cost by the input size.